src/content/advanced/egress-ip-address-azure/index.md

```diff
@@ -10,29 +10,28 @@ owner:
   - https://github.com/orgs/giantswarm/teams/team-celestial
 last_review_date: 2021-07-13
 user_questions:
-- How can I customize the public IP address for egress traffic?
+  - How can I customize the public IP address for egress traffic on Azure?
 ---
 
-# Egress IP address on Azure
+# Setting an egress IP address on Azure
 
-Giant Swarm's `Workload Clusters` use [Azure NAT Gateways](https://docs.microsoft.com/en-us/azure/virtual-network/nat-gateway/nat-overview?ocid=AID754288&wt.mc_id=azfr-c9-scottha,CFID0658) to provide internet access to node pool nodes.
+Giant Swarm's workload clusters on Azure use [NAT gateways](https://docs.microsoft.com/en-us/azure/virtual-network/nat-gateway/nat-overview) to provide internet access to worker nodes.
 
-By default a new dedicated Public IPv4 address is created during the `Workload Cluster` set up process to be used as the public
-IP address for the NAT Gateway of Node Pools' nodes.
-This means there is no way to know in advance what public IP address a certain `Workload Cluster` will have for egress
+By default, a new dedicated [public IPv4 address](https://docs.microsoft.com/en-us/azure/virtual-network/public-ip-addresses) is created during the workload cluster setup process to be used as the public
+IP address for the NAT gateway of worker nodes.
+This means there is no way to know in advance what public IP address a certain workload cluster will have for egress traffic
 until it is created.
 
-## Reuse of an existing Public IP address
+## Reuse of an existing public IP address
 
-Starting from Giant Swarm release 15.1.0 for Azure it is possible to use an existing, externally created Public IPv4 address for
-the NAT gateway of Node Pools' Nodes.
+Starting from Giant Swarm release v15.1.0 for Azure it is possible to use an existing, externally created public IPv4 address for
+the NAT gateway of worker nodes.
 
 In order to do so:
 
-1. Create a Public IPv4 Address in the same subscription the `Workload Cluster` will be created into. The IP Address can
-be in any `Resource Group`, but it needs to use the `Standard` SKU and be in the same Region as the `Workload Cluster`.
-2. Set the `Resource ID` of the Public IP Address as a value for the `giantswarm.io/workers-egress-external-public-ip`
-annotation in the `AzureCluster` `Custom Resource` as in the following example:
+1. Create a public IPv4 address in the same subscription the workload cluster will be created into. The IP Address can
+be in any resource group, but it needs to use the `Standard` SKU and be in the same region as the workload cluster.
+2. Edit the workload cluster's [`AzureCluster`]({{< relref "/ui-api/management-api/crd/azureclusters.infrastructure.cluster.x-k8s.io.md" >}}) resource via the Management API and set the resource ID of the public IP address as a value for the `giantswarm.io/workers-egress-external-public-ip` annotation, as in the following example:
```

Comment:
- Adding a link to the CRD reference
- Changing the sentence structure to make it easier to understand